main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"db"
+	"register"
+
+	"github.com/jmoiron/sqlx"
+
+	"github.com/gin-gonic/gin"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+func main() {
+	sqlDb := db.NewSql()
+	defer sqlDb.Close()
+
+	r := setupRouter(sqlDb)
+
+	_ = r.Run(":8080")
+}
+
+func setupRouter(sqlDb *sqlx.DB) *gin.Engine {
+	router := gin.Default()
+
+	jobsRepo := register.New(sqlDb)
+	adminRepo := register.NewCon(sqlDb)
+
+	router.POST("/api/v1/register/userInsert", jobsRepo.UserInsert)
```

Comment:
Read best practices for API endpoint naming and update endpoints according to it